src/YoutubeAtom.stories.tsx

```diff
@@ -24,6 +25,7 @@ export const DefaultStory = (): JSX.Element => {
                     (e) => console.log(`analytics event ${e} called`),
                 ]}
                 duration={252}
+                pillar={2} // Sport
```

Comment:
If you `import { Pillar } from '@guardian/types/Format';` into this file then you can replace this with `pillar={Pillar.Sport}` so you won't need the comment and things will carry on working if the underlying numbers change

Reply:
Oh useful, I'll change that